core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -1617,20 +1618,20 @@ public Value getTopValue() {
     }
 
     @Override
-    public Key getFirstKey() throws IOException {
-      var rfk = reader.getFirstKey();
-      if (fence.beforeStartKey(rfk)) {
-        return fencedStartKey;
+    public Text getFirstRow() throws IOException {
+      var rfk = reader.getFirstRow();
+      if (fence.beforeStartRow(rfk)) {
```

Comment:
It seems that getFirstRow() can return null and passing null to beforeStartRow or beforeStartKey may cause an NPE.  This was an existing problem.  Not sure when or if it would happen.

Reply:
I will add a null check for these methods